docs/world-compat-details/index.md

```diff
@@ -10,3 +10,220 @@ sidebar_position: 4
 已知的兼容方案有：
 
 * [`libLoL`](liblol.md)
+
+# 导言
```

Comment:
Either make all headings one level deeper (i.e. h1 → h2, etc.), or move the content into separate article(s)? Because a Markdown document should in general only contain one h1.